core/src/wsv.rs

```diff
@@ -173,6 +173,20 @@ impl<W: WorldTrait> WorldStateView<W> {
             .collect()
     }
 
+    /// Calculates network TPS.
+    ///
+    /// `None` means that transactions can not yet be calculated.
+    #[allow(clippy::float_arithmetic, clippy::cast_precision_loss)]
+    pub fn transactions_per_second(&self) -> Option<f64> {
+        let start_time_ms = self.blocks.iter().next()?.as_inner_v1().header.timestamp;
+        let end_time_ms = current_time().as_millis();
```

Comment:
Have we considered the time crate? Or Chrono?

Reply:
Normally, `std::time` doesn't offer good-enough precision for times in the second-to-minute range. Both `time` and its successor `chrono` do things a little more precisely. I think that since the block time-stamp is set using the same `current_time()` this either doesn't matter at all, or deserves its own issue.